### PR TITLE
add links and descriptions to available data sets.

### PR DESCRIPTION
**Pull request by staffanru**
Adds links and descriptions to the three available data sets.

Only adding links to clarin and openslr since it hasn't released on LDC yet.

### CODE REVIEW

File: public/static/locales/eng/the-database.json [hunk 1]
```diff
@@ -7,7 +7,11 @@
...
-        "end": "The first release of 100,000 utterances has been released on <2>OpenSLR</2>. Subscribe to the newsletter below to be notified when further datasets becomes accessible."
+        "preface-releases": "There are currently three releases of data sets from Samrómur.",
+        "samromur-21-05": "This is the first release of the data from the Samrómur collection. The corpus contains 100,000 (145 hours) validated speech-recordings in Icelandic, available both on <2>Clarin</2> and <6>OpenSLR</6>",
+        "samromur-children": "This release of data from the Samrómur collection focuses on children (4-17 years old). It contains more than 137.000 (131 hours) of validated speech recordings uttered by children in Icelandic, available both on <2>Clarin</2> and <6>OpenSLR</6>.",
+        "samromur-queries": "This release of data from the Samrómur collection focuses on queries. It contains 17,475 (20 hours) of validated speech-recordings in Icelandic,  available both on <2>Clarin</2> and <6>OpenSLR</6>.",
+        "sign-up-for-updates": "Subscribe to the newsletter below to be notified when further datasets becomes accessible."
```
[the remark this answers is not in the document]
Comment (judyfong) [reply public/static/locales/eng/the-database.json:14]: David said he would but I don't think he ever did.

File: src/components/dataset/about.tsx [hunk 2]
```diff
@@ -92,21 +98,91 @@ export const AboutDataset: React.FunctionComponent<Props> = ({
...
+                    </Bold>
+                    <Trans i18nKey={'database.samromur-children'} t={t}>
+                        Þessi útgáfa gagna úr safni Samróms einblínir á börn
+                        (4-17 ára). Útgáfan inniheldur 137.000 (131 klst.)
+                        staðfestar talupptökur frá börnum á íslensku, fáanlegar
+                        bæði á{' '}
+                        <StyledLink
+                            target={'blank'}
+                            href={
+                                'https://repository.clarin.is/repository/xmlui/handle/20.500.12537/185'
```
Comment (judyfong) [inline src/components/dataset/about.tsx:142]: same with this link

File: src/components/dataset/about.tsx [hunk 3]
```diff
@@ -92,21 +98,91 @@ export const AboutDataset: React.FunctionComponent<Props> = ({
...
+                        <br />
+                    </Bold>
+                    <Trans i18nKey={'database.samromur-queries'} t={t}>
+                        Þessi útgáfa gagna úr safni Samróms einblínir á
+                        fyrirspurnir. Útgáfan inniheldur 17.475 (20 klst.)
+                        staðfestar talupptökur á íslensku, fáanlegar bæði á{' '}
+                        <StyledLink
+                            target={'blank'}
+                            href={
+                                'https://repository.clarin.is/repository/xmlui/handle/20.500.12537/180'
```
Comment (judyfong) [inline src/components/dataset/about.tsx:169]: and finally this one

File: public/static/locales/eng/the-database.json [hunk 4]
```diff
@@ -7,7 +7,11 @@
         "title": "The Samrómur database",
         "beginning": "Samrómur is an open and accessible database of voices that everyone is free to use when developing software in Icelandic.",
         "middle": "The database consists of sentences and audio clips from the reading of those sentences as well as metadata. Each entry in the database contains FLAC audio clips and the corresponding text file.",
-        "end": "The first release of 100,000 utterances has been released on <2>OpenSLR</2>. Subscribe to the newsletter below to be notified when further datasets becomes accessible."
+        "preface-releases": "There are currently three releases of data sets from Samrómur.",
```
Comment (judyfong) [inline public/static/locales/eng/the-database.json:10]: on line ten you call them data sets but on line 14 you call them datasets.

**Review by judyfong — CHANGES_REQUESTED** [review]
Please make the changes. Looks good overall.

File: src/components/dataset/about.tsx [hunk 5]
```diff
@@ -92,21 +98,91 @@ export const AboutDataset: React.FunctionComponent<Props> = ({
...
+                        <br />
+                    </Bold>
+                    <Trans i18nKey={'database.samromur-21-05'} t={t}>
+                        Þetta er fyrsta útgáfan af gögnum úr safni Samróms.
+                        Útgáfan inniheldur 100.000 (145 klst.) staðfestar
+                        talupptökur á íslensku, fáanlegar bæði ás{' '}
+                        <StyledLink
+                            target={'blank'}
+                            href={
+                                'https://repository.clarin.is/repository/xmlui/handle/20.500.12537/189'
```
[the remark this answers is not in the document]
Comment (staffanru) [reply src/components/dataset/about.tsx:114]: done

File: src/components/dataset/about.tsx (hunk 2 above)
Comment (staffanru) [reply src/components/dataset/about.tsx:142]: done

File: src/components/dataset/about.tsx (hunk 3 above)
Comment (staffanru) [reply src/components/dataset/about.tsx:169]: done

File: public/static/locales/eng/the-database.json (hunk 4 above)
Comment (staffanru) [reply public/static/locales/eng/the-database.json:10]: going for data sets in both

File: public/static/locales/eng/the-database.json [hunk 6]
```diff
@@ -7,7 +7,11 @@
...
+        "samromur-21-05": "This is the first release of the data from the Samrómur collection. The corpus contains 100,000 (145 hours) validated speech-recordings in Icelandic, available both on <2>Clarin</2> and <6>OpenSLR</6>",
+        "samromur-children": "This release of data from the Samrómur collection focuses on children (4-17 years old). It contains more than 137.000 (131 hours) of validated speech recordings uttered by children in Icelandic, available both on <2>Clarin</2> and <6>OpenSLR</6>.",
```
[the remark this answers is not in the document]
Comment (staffanru) [reply public/static/locales/eng/the-database.json:12]: fixed

**Review by judyfong — APPROVED** [review]
Perfect

**Pull request merged**